docs/docs/cmd/teams/tab/tab-list.md

```diff
@@ -37,3 +37,46 @@ Include all the values from the tab configuration and associated teams app
 ```sh
 m365 teams tab list --teamId 00000000-0000-0000-0000-000000000000 --channelId 19:00000000000000000000000000000000@thread.skype --output json
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    [
+      {
+        "id": "34991fbf-59f4-48d9-b094-b9d64d550e23",
+        "displayName": "Polly",
```

Comment:
Let's try to align the output with the data used in the example. Same goes for text and csv.